### PR TITLE
[DOCS] Updates machine learning custom URL screenshots

### PR DESCRIPTION
**Pull request by lcawl**
This PR updates the documentation and screenshots related to custom URL enhancements.

### CODE REVIEW

**Comment by lcawl** [comment]
You can preview these documentation changes here: https://doc-review-63bf6.firebaseapp.com/x2/ml-configuring-url.html

File: x-pack/docs/en/ml/customurl.asciidoc [hunk 1]
```diff
@@ -1,22 +1,48 @@
...
+As in this case, the custom URL can contain 
+<<ml-configuring-url-strings,dollar sign delimited tokens>>, which 
+are populated when you click the link in the anomalies table. If you were 
+interested in the following anomaly, for example: 
 
 [role="screenshot"]
-image::images/ml-customurl.jpg["Links in the Anomaly Explorer anoamilies table"]
+image::images/ml-customurl.jpg["An example of the custom URL links in the Anomaly Explorer anomalies table"]
+
+...clicking `My Link 1` opens the *Discover* page and show results for the
```
Comment (peteharverson) [inline x-pack/docs/en/ml/customurl.asciidoc]: Should be `shows` in `and shows results...`

File: x-pack/docs/en/ml/customurl.asciidoc [hunk 2]
```diff
@@ -1,22 +1,48 @@
...
 
 When you create an advanced job or edit any job in {kib}, you can optionally
-attach one or more custom URLs. You can also specify these custom settings when
-you create or update jobs by using the {ml} APIs.
+attach one or more custom URLs. 
 
-The custom URLs provide links from the anomalies table in the Anomaly Explorer
-or Single Metric Viewer window in {kib} to custom dashboards or external
+The custom URLs provide links from the anomalies table in the *Anomaly Explorer*
+or *Single Metric Viewer* window in {kib} to custom dashboards or external
```
Comment (peteharverson) [inline x-pack/docs/en/ml/customurl.asciidoc]: Is it worth adding the Discover page to this sentence? e.g. `... links from the anomalies table in the Anomaly Explorer or Single Metric Viewer to Kibana dashboards, the Discover page or external websites` ?

**Review by peteharverson — COMMENTED** [review]
Is it worth adding a brief section on the new Time Range control, which allows the user to drill down to a fixed time interval either side of the anomalous bucket? Note the new time range setting has resulted in an extra `time_range` attribute to the custom URL JSON alongside the existing `url_name` and `url_value` fields, which are shown in the screenshot in the String substitution section.

**Comment by lcawl** [comment]
Thanks, @peteharverson , I've added information about the time range options.  If this wasn't what you had in mind or you have additional suggestions, let me know!

**Review by peteharverson — APPROVED** [review]
LGTM

**Pull request merged**